src/main/java/org/grouplens/grapht/Instantiators.java

```diff
@@ -88,6 +98,8 @@ public static Instantiator ofProviderInstantiator(Instantiator pinst) {
         // First try to unpack the instantiator
         if (instantiator instanceof ProviderInstantiator) {
             Instantiator itor = ((ProviderInstantiator) instantiator).providerInstantiator;
+            // ---  toProvider instance push to MDC ---------------------------------------
+            mdcContext.put(instantiator.getClass().toString(), itor.toString());
```

Comment:
We don't need to put things in the context here - all this is doing is wrapping an instantiator. We're wanting to log when instantiators are activated.